Moonshot/room_templates/LiftButton.gd

```diff
@@ -0,0 +1,18 @@
+extends Sprite
+
+var red = load("res://assets/environment/LiftButtonRed.png")
+var green = load("res://assets/environment/LiftButtonGreen.png")
+var on = load("res://assets/environment/LiftButtonON.png")
+
+func _ready():
+	set_texture(red)
+
+func change_texture(state):
```

Comment:
you could set state as being `LiftButtonRed `, `LiftButtonGreen `, `LiftButtonON `.

then this could become:

```gdscript
func change_texture(state):
   return load("res://assets/environment/" + state + ".png")

```

Reply:
Or 

```gdscript
var state_dictionary: Dictionary = {
  "red": load("res://assets/environment/LiftButtonRed.png"),
  "green": load("res://assets/environment/LiftButtonGreen.png"),
  "on": load("res://assets/environment/LiftButtonON.png")
}
```

and index it by a key

Reply:
I'll nab that dict, ty.

Reply:
Resolved https://github.com/kepler471/moonshot/pull/231/commits/c2e7d1e6ed38100866d34c40563f0ca7c066f919